package.json

```diff
@@ -2000,6 +2000,12 @@
                     "default": false,
                     "description": "Add PYTHONNOUSERSITE to kernels before starting. This prevents global/user site-packages from being used in the PYTHONPATH of the kernel.",
                     "scope": "machine"
+                },
+                "jupyter.enablePreciseKernelCompletions": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Enables the use of Jedi to generate more precise kernel completions. This can impact peformance, such as preventing executions from running. Use with caution.",
```

Comment:
I think this works, but question. Do most users even have a clue what Jedi is?

Reply:
Actually this is probably good as is, at least they could look it up.

Reply:
Possible alternate name / wording:

enableExtendedKernelCompletions - "Turns on the Jedi language server to provide extended IntelliSense completions for running Jupyter notebooks. This can greatly impact notebook cell execution performance. Use with caution."

Honestly not sure if that's better so take it or leave it :). Kinda liked extended over precise since precise sounded more like numerically getting closer to a target.

Reply:
I like 'extended' too but didn't like the use of Jedi language server. It's not a jedi language server. It's internal to the kernel.  Maybe I should mention the config setting this changes.